### ISSUE TITLE
amp 导致训练时报错 ，提示GPU算力低

### ISSUE THREAD

**Issue by sworderzjf1** [PaddlePaddle/Paddle#51849]
### 请提出你的问题 Please ask your question

用paddle训练时，报错退出，具体报错信息如下：
auto_cast.py:328: UserWarning: For float16, amp only support NVIDIA GPU with Compute Capability 7.0 or higher, current GPU is: NVIDIA GeForce GTX 950M, with Compute Capability: 5.0.

**Comment by wuyefeilin** [comment]
Compute Capability 7.0 or higher , 你目前只有5.0, 所以跑不了gpu版本。 如果你只是上手练习的话，可以试试cpu版本

**Comment by wuyefeilin** [comment]
Compute Capability 7.0 or higher , 你目前只有5.0, 所以跑不了float16哈

**Comment by sworderzjf1** [comment]
谢谢！麻烦给予更加详细的建议和指导，我知道跑不了float16，但是怎么关闭amp呢？以下代码是从paddle网站下载的demo代码，默认的_amp_level 都是O0，但就是会由amp功能调用auto_cast.py，继而导致报float16的错误。

# 模型组网并初始化网络
lenet = paddle.vision.models.LeNet(num_classes=10)
model = paddle.Model(lenet)

# 模型训练的配置准备，准备损失函数，优化器和评价指标
model.prepare(paddle.optimizer.Adam(parameters=model.parameters()),
              paddle.nn.CrossEntropyLoss(),
              paddle.metric.Accuracy())

# 模型训练
model.fit(train_dataset, epochs=5, batch_size=64, verbose=1)

**Comment by wuyefeilin** [comment]
参考5.1的训练，直接构造训练流程看看哈https://www.paddlepaddle.org.cn/documentation/docs/zh/practices/cv/image_classification.html#moxingxunlian
